components/esp_lvgl_port/src/lvgl9/esp_lvgl_port.c

```diff
@@ -0,0 +1,198 @@
+/*
```

Comment:
esp_lvgl_port.c for LVGL8 and 9 is almost the same. Would it make sense to combine some features into common file?

It would help us track bugfixes in the future a lot

Reply:
Now, the files are same. But in future, in LVGL9 will be more optimization, task sleeping and similar...